HandLandmark[] landmarks    # Array of hand landmarks
